app/workers/scheduler/feed_cleanup_scheduler.rb

```diff
@@ -5,18 +5,36 @@ class Scheduler::FeedCleanupScheduler
   include Sidekiq::Worker
 
   def perform
+    reblogged_id_sets = {}
+    feedmanager = FeedManager.instance
```

Comment:
FeedManager -> feed_manager

But since we're in a hurry I'll clean it up later, it's not substantial